### PR TITLE
Add support for symfony form help

### PR DESCRIPTION
**Pull request by core23**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Add support for symfony form help
```

### CODE REVIEW

**Comment by core23** [comment]
`form_help` exists in symfony 4.3:
https://symfony.com/doc/4.3/form/form_customization.html#form-help-form-view

**Comment by OskarStark** [comment]
Anything for the docs?

**Comment by core23** [comment]
> Anything for the docs?

No, it's the expected behaviour you know for symfony forms.

**Comment by franmomu** [comment]
I think you should also add the call in our [`form_row` definition](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/views/Form/form_admin_fields.html.twig#L348) since we override the `form_row` block from [form_div_layout.html.twig](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig#L343).

Should we deprecate `sonata_help`? or they do different things?

File: src/Resources/views/Form/form_admin_fields.html.twig [hunk 1]
```diff
@@ -395,6 +401,8 @@ file that was distributed with this source code.
             {% if sonata_admin is defined and sonata_admin_enabled and sonata_admin.field_description.help|default(false) %}
                 <span class="help-block sonata-ba-field-help">{{ sonata_admin.field_description.help|trans({}, sonata_admin.field_description.translationDomain ?: admin.translationDomain)|raw }}</span>
             {% endif %}
+
+            {{ form_help(form) }}
```
Comment (core23) [inline src/Resources/views/Form/form_admin_fields.html.twig]: Here is the call @franmomu. GitHub messed up the code folding

**Comment by core23** [comment]
> Should we deprecate `sonata_help`? or they do different things?

Yes. There is no need for this. Can you do this @franmomu

File: src/Resources/views/Form/form_admin_fields.html.twig (hunk 1 above)
Comment (franmomu) [reply src/Resources/views/Form/form_admin_fields.html.twig]: I saw it just after press Comment 😬

**Pull request merged**